my-app/src/components/productpage/filter/filterHeader/FilterHeader.jsx

```diff
@@ -1,11 +1,14 @@
 import React from "react";
-
+import { ACTION_TYPE } from "../../../../store/Actions";
+import {useFilter} from "../../../../store/data/FilterContext"
 function FilterHeader() {
+  const {filterDispatch} =useFilter();
+
   return (
     <div>
         <div class="filter-item filter-header">
           <h2>Filters</h2>
-          <button class="btn btn-s btn-link">clear</button>
+          <button class="btn btn-s btn-link" onClick={()=>filterDispatch({type:ACTION_TYPE.RESET})}>clear</button>
```

Comment:
You can make a handler function for this since you have done similar way at other places. The code will be consistent.

Reply:
sure will do that